rust/worker/src/execution/operators/knn.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Debug)]
+pub struct KnnOperator {
+    pub embedding: Vec<f32>,
+    pub fetch: u32,
```

Comment:
nit: really think we should propagate language from the user facing api down - we should refrain from switching words internally unless needed

Reply:
Will update the language in the future